src/sources.js

```diff
@@ -195,18 +195,27 @@ class CommunitySource extends Source {
 
     if (!this.repoName) throw new Error(`Cannot construct a ${this.constructor.name} without a repoName after splitting on /@/`);
     if (!this.branch) {
-      this.branch = "master";
+      this.branch = fetch(`https://api.github.com/repos/${this.owner}/${this.repoName}`)
+        .then((res) => res.json())
+        .then((data) => data.default_branch || "master")
+        .catch(() => {
+          console.log(`Error interpreting the default branch of ${this.constructor.name}`);
+          return "master";
+        });
     }
   }
 
   static get _name() { return "community"; }
   get repo() { return `${this.owner}/${this.repoName}`; }
-  get baseUrl() { return `https://github.com/${this.repo}/raw/${this.branch}/`; }
+  async baseUrl() {
+    return `https://github.com/${this.repo}/raw/${await this.branch}/`;
+  }
 
-  get repoNameWithBranch() {
+  async repoNameWithBranch() {
+    const branch = await this.branch;
     return this.branch === "master"
       ? this.repoName
-      : `${this.repoName}@${this.branch}`;
+      : `${this.repoName}@${branch}`;
```

Comment:
This change (in the first commit on the branch) forgot to update the `this.branch === "master"` comparison to use `branch` instead (the local variable after `await`-ing), which was the source of

> however there is a small bug:
> 
> The `getAvailable` API response includes the `@branch` syntax, even for the default branch.

Reply:
That's where the bug was, but since we didn't keep track of the default branch we couldn't check whether the  branch (`await this.branch`) was the default one.

Reply:
Yep, I understand that. IMO, the bug in the first commit should be fixed, regardless of the subsequent change to stop assuming `master` is the default branch for a repo.

Reply:
Resolved in a2ecf7c